src/qililab/platform/platform.py

```diff
@@ -893,9 +893,10 @@ def _execute_qblox_compilation_output(self, output: QbloxCompilationOutput, debu
         results = QProgramResults()
         for bus_alias, bus in buses.items():
             if bus.has_adc():
-                bus_results = bus.acquire_qprogram_results(acquisitions=acquisitions[bus_alias])
-                for bus_result in bus_results:
-                    results.append_result(bus=bus_alias, result=bus_result)
+                for channel in buses[bus.alias].channels:
```

Comment:
we need to modify the unittest for this to pass.

Reply:
The unitests were left intact and all passed, the problem was a missing condition to exclude all non qblox modules